scripts/_infra_k8s.py

```
@@ -0,0 +1,349 @@
+    stdout: str
+    stderr: str
+    return_code: int
+
+
+def mask_secret(value: str, stream: cabc.Callable[[str], None] = print) -> None:
+    """Emit GitHub Action secret masking command."""
+    if value:
+        stream(f"::add-mask::{value}")
+
+
+def parse_bool(value: str | None, default: bool = True) -> bool:
```

Ruff (0.14.13)

Boolean-typed positional argument in function definition

(FBT001)